abilities/defensive-evasion/3864fd22-5c63-41c9-bdbc-a66b5ffa3f5e.yml

```diff
@@ -10,5 +10,6 @@
   executors:
     windows:
       command: |
-        $shell = New-Object -ComObject Wscript.Shell
-        Set-ExecutionPolicy Bypass | echo $shell.sendkeys("Y`r`n")
```

Comment:
can you explain this change set? can we verify it works on windows 7 and 10?

Reply:
works on Windows7 and Windows10